src/test/java/uk/gov/hmcts/reform/cosapi/util/TestFileUtil.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.cosapi.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@SuppressWarnings("PMD")
+public class TestFileUtil {
+
+    private TestFileUtil() {
+    }
+
+    public static String loadJson(final String filePath) throws Exception {
+        return new String(loadResource(filePath), Charset.forName("utf-8"));
+    }
+
+    public static byte[] loadResource(final String filePath) throws Exception {
+        URL url = TestFileUtil.class.getClassLoader().getResource(filePath);
+
+        if (url == null) {
+            throw new IllegalArgumentException(String.format("Could not find resource in path %s", filePath));
+        }
```

Comment:
what is the difference between this class and TestResouceUtil?

Reply:
Removed. Used by rebase ISDB-244.